docs/admin/admin-schema.yaml

```diff
@@ -12,6 +12,16 @@ definitions:
           type: string
         email:
           type: string
+        active:
+          type: boolean
+        blocked:
+          type: boolean
+        dateOfInvitation:
```

Comment:
should be type: string, format: date-time

https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#data-types